src/hest/HESTData.py

```diff
@@ -12,6 +12,9 @@
 from hestcore.wsi import (WSI, CucimWarningSingleton, NumpyWSI,
                           contours_to_img, wsi_factory)
 from loguru import logger
+from spatialdata import SpatialData
```

Comment:
We need to move this import line inside the to_spatial_data() method, the SpatialData is huge

Reply:
Moved in [98d0acf](https://github.com/mahmoodlab/HEST/pull/63/commits/98d0acfb87a8ecec1f5dca5fdee00032735900a7) - I had it outside to enable the type hint in the function, but appreciate that it comes with a lot of overhead